docs/criteria/accelerators/alpha.md

```diff
@@ -15,4 +15,30 @@ All criteria must be met for the accelerator to be considered Alpha, with except
 
 ## Alpha Release Criteria
 
-TBD
+The Alpha Release Criteria for accelerators is set at a level that ensures the Accelerator operates in common conditions with a low error rate.
+
+Accelerators are intended to be more stable than Connectors, and as such the Alpha criteria is set slightly higher than that of Connectors.
+
+The Alpha Release Criteria is not intended to cover any edge cases or complex functionality, so federation, streaming. However, some testing is required to cover TPC-H derived queries.
+
+### All Accelerators
+
+- [ ] The accelerator implements the querying functionality of the native accelerator source in all [Access Modes](../definitions.md)
+- [ ] The accelerator supports only String, Number, Date and Binary [Core Connector Data Types](../definitions.md).
+- [ ] The accelerator executes common use case queries with a low error rate (e.g. `SELECT * FROM tbl WHERE col = 'blah'`).
+- [ ] Known [Minor and Major](../definitions.md) bugs are logged, but not required to be fixed unless needed to achieve a low query error rate and/or TPC-H success.
+  - A "low error rate" indicates that more than 90% of the time, the common use case succeeds.
```

Comment:
It's not clear to me why we would have any error rate for a given query - either it works or doesn't. Is this a load related metric or something else?

Reply:
Or is this saying that for some hypothetic set of common use case queries, over 90% of them work?

Reply:
> It's not clear to me why we would have any error rate for a given query - either it works or doesn't. Is this a load related metric or something else?

It probably doesn't apply to Accelerators. I included it for Connectors because the set of common use cases can vary by connector, but for accelerators the common use cases are more tightly scoped (and more importantly, usually the same across accelerators). I've removed this item and add an additional item to common use cases to be more tightly scoped.